### PR TITLE
W-15289109: remove usage of internals from Mule APIs

### PR DESCRIPTION
**Pull request by rbourbonnavarro**

### CODE REVIEW

File: src/main/java/module-info.java [hunk 1]
```diff
@@ -124,45 +124,13 @@
...
-      org.mule.runtime.extensions.xml.support,
-      org.mule.runtime.dsl.api,
-      org.mule.runtime.properties.config,
-      org.mule.runtime.metadata.support,
-      org.mule.runtime.spring.config,
-      org.mule.runtime.tls,
-      org.mule.test.runner,
-      com.mulesoft.mule.runtime.ee.extension.model,
-      com.mulesoft.mule.runtime.cluster,
       com.mulesoft.anypoint.gw.module.autodiscovery;
```
Comment (elrodro83) [inline src/main/java/module-info.java:127]: make the corresponding change in the gateway

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Review by elrodro83 — APPROVED** [review]

**Comment by rbourbonnavarro** [comment]
--validate

**Comment by rbourbonnavarro** [comment]
--validate

**Comment by rbourbonnavarro** [comment]
--validate

**Review by elrodro83 — APPROVED** [review]

**Pull request merged**